src/web/app/pages-student/student-home-page/student-home-page.component.html

```diff
@@ -1,3 +1,61 @@
-<p>
-  student-home-page works!
-</p>
+<br>
+<br>
+<ng-container *ngFor="let course of courses">
+  <div class="card bg-light top-padded" *ngIf="course.sessions.length">
```

Comment:
There should be something else shown when a course does not have any sessions, right?